### PR TITLE
Copy review pt1

### PR DESCRIPTION
**Pull request by tdg**
Copy review pt1 partly addresses #561

### CODE REVIEW

**Comment by tdg** [comment]
Please could you review @joshr @baob @aspiers?
I'll continue with ptii asap

File: app/views/static_pages/faq.html.haml [hunk 1]
```diff
@@ -91,9 +94,46 @@
...
+
+    %p
+
+      No.
+    %p
+      The Swap My Vote team is committed to creating a tool that can help make votes fairer and more equal.
+      We believe in a healthy democracy with multiple parties, and a better voting system to gauge the public's support for them.
+      The platform is independent of any party and will remain so.
+
+    %a(name="parties")
```
[the remark this answers is not in the document]
Comment (aspiers) [reply app/views/static_pages/faq.html.haml:110]: I disagree with the hound dog on this one.

File: app/views/static_pages/faq.html.haml [hunk 2]
```diff
@@ -67,6 +55,17 @@
       community amongst voters, all hoping to make the country run as
       well as possible for a better future for everyone.
 
+    -
```
[the remark this answers is not in the document]
Comment (aspiers) [reply app/views/static_pages/faq.html.haml:58]: @tdg This doesn't look right.

**Review by aspiers — CHANGES_REQUESTED** [review]
Mostly looks great; see the few comments from hound and me.

**Review by baob — DISMISSED** [review]

**Comment by aspiers** [comment]
Rebased

**Review by aspiers — APPROVED** [review]

**Pull request merged**